### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by kind4soul** [aclap-dev/video-downloadhelper#1419] [Help and Q&A]
### Your question

Hi, I would like a refund. I purchased the licence for one website, but it can only download the audio. How do I go about this? thanks

### Addon & CoApp version and details

Hi, I would like a refund. I purchased the licence for one website, but it can only download the audio. How do I go about this? thanks

**Comment by jcfrog** [comment]
Hi, please send us an email at licenses@downloadhelper.net
